src/compas/geometry/surfaces/nurbs.py

```diff
@@ -320,20 +320,32 @@ def from_step(cls, filepath):
         return new_nurbssurface_from_step(cls, filepath)
 
     @classmethod
-    def from_fill(cls, curve1, curve2):
-        """Construct a NURBS surface from the infill between two NURBS curves.
+    def from_fill(cls, curve1, curve2, curve3=None, curve4=None, style='stretch'):
+        """Construct a NURBS surface from the infill between two, three or four contiguous NURBS curves.
 
         Parameters
         ----------
-        curve1 : :class:`~compas.geometry.NurbsCurve`
-        curve2 : :class:`~compas.geometry.NurbsCurve`
+        curve1 : :class:`~compas_occ.geometry.OCCNurbsCurve`
```

Comment:
sorry i didn't notice this before.
should be 
```
:class:`compas.geometry.NurbsCurve`
```

for all 3 curves, since this will switch between Rhino and OCC in the back...